### DISCUSSION TITLE
Support UBSan on Windows

### DISCUSSION THREAD

**Discussion by 66Leo66** [cpeditor/cpeditor#1114] [Ideas]
Currently on my machine neither `c++/g++` nor `clang++` is able to compile with `-fsanitize=undefined`.

g++ misses something
```
c:/program files (x86)/cpeditor/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/11.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cannot find -lubsan 
collect2.exe: error: ld returned 1 exit status
```

Is it possible to include the required sanitizer library with release?

**Comment by ouuan** [comment]
I'm not familiar with this, but after a quick research, it seems that ubsan is not available on Windows:

- https://stackoverflow.com/questions/43995718/how-to-use-ubsan-on-gcc-windows-8-1
- https://github.com/msys2/MINGW-packages/issues/3163
